demo/src/containers/Chat/Chat.tsx

```diff
@@ -43,19 +47,21 @@ export const Chat = () => {
   const messagesEndRef = useRef<HTMLDivElement | null>(null);
 
   useEffect(() => {
+    chatClient.logger.debug('updating getPreviousMessages useEffect', { getPreviousMessages });
     // try and fetch the messages up to attachment of the messages listener
-    if (getPreviousMessages && loading) {
+    if (getPreviousMessages) {
```

Comment:
What will happen on discontinuity here if you remove the loading state?

Reply:
Have fixed this - I've also opened https://ably.atlassian.net/browse/CHA-700 which relates